app/tasks/maintenance_tasks/csv_task.rb

```diff
@@ -3,9 +3,9 @@
 require 'csv'
 
 module MaintenanceTasks
-  # Base class that is inherited by the host application's Task classes for
+  # Module that is included into Task classes by Task#csv_collection for
   # processing CSV files.
-  class CsvTask < Task
+  module CsvTask
```

Comment:
Should this be annotated as a private API (on top of `private_constant`)?
```suggestion
  # @api private
  module CsvTask
```

Reply:
I'm not sure this module is more deserving of being denoted as private API compared to some of our other internals. However, we are planning on actually [dropping the private_constant declarations](https://github.com/Shopify/maintenance_tasks/issues/314) so that we no longer prevent users from extending our classes if they choose to. Consequently, we may want to switch to using YARD's `@api private` tag in the future anyways. But I think I'd leave this for now 😄

Reply:
I think all the private constants should be `@api private`, I fixed this one, there's a couple of others missing, I'll add them to a PR turning on documentation generation I think.

Reply:
Yeah sorry, I didn't even realize most of our constants were already denoted as `@api private` 🤦‍♀️